### PR TITLE
Testing

### LINKED ISSUE #35: No permission when trying to install pip package
When I try to install packages with pip to Yunohost JupiterLab venv environment
I get this error:

image

The folder /opt/jupyterlab/ belongs to root...

`chown -R $USER: /opt/jupyterlab` resolves the permission problem. ($USER=my login name)
Would it make sense to set the permission for the connected user  as default?

### LINKED ISSUE #45: Not able install extensions (requirex nodjejs >= 10)
I am not sure if extensions are supported, if it is then I think nodejs package needs to be updated as I am getting below error,

image

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *#35*
- *#45*

## Solution
- *Fix #35 (admin can now install packages with pip)*
- *Fix #45*
- *Upgrade node version*
- *Upgrade to 2.1.1*

Thank you @ericgaspar 

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/jupyterlab_ynh%20PR43%20(Kay0u)/)

### CODE REVIEW

File: scripts/restore [hunk 1]
```diff
@@ -62,8 +62,8 @@ ynh_restore_file --origin_path="$final_path"
 #=================================================
 
 # Restore permissions on app files
-chown -R root: $final_path
-chown -R $admin: $final_path/.venv/share
+chown -R $admin: $final_path/
+chmod -R 755 $final_path/.venv/
```
Comment (ericgaspar) [inline scripts/restore:66]: I forgot to change this one

File: scripts/upgrade [hunk 2]
```diff
@@ -170,8 +170,8 @@ ynh_add_systemd_config
 #=================================================
 
 # Set permissions on app files
-chown -R root: $final_path
-chown -R $admin: $final_path/.venv/share
+chown -R $admin: $final_path/
+chmod -R 755 $final_path/.venv/
```
Comment (ericgaspar) [inline scripts/upgrade:174]: ...and this one.

**Comment by ericgaspar** [comment]
I have problems installing **matplotlib** :/  it needs permission to the `/bin` folder...

**Comment by kay0u** [comment]
I think everything is working now I'll wait your feedback before merging :-)

**Comment by ericgaspar** [comment]
It works for me, Thanks for your help :)

**Pull request merged**